client/webserver/webserver_test.go

```diff
@@ -396,7 +403,9 @@ func TestAPIWithdraw(t *testing.T) {
 		return resp.OK
 	}
 
-	body = &withdrawForm{}
+	body = &withdrawForm{
```

Comment:
This is needed because `resolvePass` will return an error if this is an empty string.